contracts/libraries/SafeToL2Migration.sol

```diff
@@ -100,7 +96,7 @@ contract SafeToL2Migration is SafeStorage {
             "", // We cannot detect signatures
```

Comment:
We need to add `payable` to the line above right?

Reply:
🤦🏾 Updated 👍🏾